WooCommerce/Classes/POS/Models/PointOfSaleAggregateModel.swift

```diff
@@ -24,7 +24,9 @@ protocol PointOfSaleAggregateModelProtocol {
     func cancelCardPaymentsOnboarding()
     func trackCardPaymentsOnboardingShown()
 
-    var currentViewState: ItemsViewState { get }
+    var itemsViewState: ItemsViewState { get }
```

Comment:
Should we rename `itemsViewState` to `productsViewState`, and `itemsController` to `productsController`? So it clears up the specialization of each item when exposed:

```
var productsViewState: ItemsViewState { productsController.itemsViewState }
var purchasableItemsSearchViewState: ItemsViewState { purchasableItemsSearchController.itemsViewState }
var couponsViewState: ItemsViewState { couponsController.itemsViewState }

[...]

private let productsController: PointOfSaleItemsControllerProtocol
private let purchasableItemsSearchController: PointOfSaleSearchingItemsControllerProtocol
private let couponsController: PointOfSaleCouponsControllerProtocol
```

Reply:
I'll look at this when I do the controllers